net/libfab/libfab.c

```diff
@@ -560,39 +617,12 @@ static void libfab_poller(struct m0_fab__tm *tm)
 
 		m0_tl_for(m0_nep, &tm->ftm_net_ma->ntm_end_points, net) {
 			xep = libfab_ep_net(net);
-			aep = (xep->fep_listen == NULL) ? xep->fep_send :
-			      xep->fep_listen->pep_tx_ep;
+			aep = (xep->fep_listen == NULL) ? xep->fep_aep :
+			       xep->fep_listen->pep_aep;
```

Comment:
I propose to define an inline function for this, because same code is duplicated in multiple places.

Reply:
Done